pkgs/tools/security/pcsc-scm-scl011/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "SCM Microsystems SCL011 chipcard reader user space driver";
-    homepage = "http://www.scm-pc-card.de/index.php?lang=en&page=product&function=show_product&product_id=630";
+    homepage = "https://www.scm-pc-card.de/index.php?lang=enhttp://www.scm-pc-card.de/index.php?lang=en&page=product&function=show_product&product_id=630page=producthttp://www.scm-pc-card.de/index.php?lang=en&page=product&function=show_product&product_id=630function=show_producthttp://www.scm-pc-card.de/index.php?lang=en&page=product&function=show_product&product_id=630product_id=630";
```

Comment:
I think it failed here

Reply:
Huh, probably an oddity of the particular script used. Fixed it manually